# Diet-variability-in-SRKW-and-SRKW
This repo includes the code and metadata used to generate Van Cise et al. *in prep*, which is a comparison of variability in diet composition across two populations of resident killer whales in the North Pacific: southern resident killer whales and southern Alaska resident killer whales. The following information outlines the use of the scripts in this repo.

### Manuscript

**Oorc_diet_comparative_analysis.Rmd:** The manuscript itself was generated in [RMarkdown](https://rmarkdown.rstudio.com/). This script calls data outputs generated by the data analysis scripts outlined below.

### dada2 pipeline

We used the [dada2](https://benjjneb.github.io/dada2/) pipeline to perform initial quality control of raw sequence data output as fastq files from the MiSeq sequencer.

**dada2QAQC.R:** This file runs the dada2 pipeline, using the general format recommend in the [dada2 tutorial](https://benjjneb.github.io/dada2/tutorial.html), with appropriate parameters for the present dataset. Major steps in this pipeline include:

* Filter and trim raw read sequences based on a minimum quality score of 30
* Merge filtered forward and reverse reads
* De-replicate merged reads and remove chimeras
* Infer ASVs from sequences
* Assign taxonomic classification to ASVs using a naive Bayesian classifier implemented in dada2 and a custom-generated reference file (**NPac_fishes_and_sharks.fasta**).
* Export read count per ASV per sample, taxonomic classification for each ASV, and a table tracking number of reads per sample at each QAQC step.

*Note: anyone using code from this script should be careful to ensure that they specify parameters appropriate for their own data. If you publish data using code from these scripts, cite [dada2](https://benjjneb.github.io/dada2/).*

**Rundada2.slurm:** It's ideal to run dada2 on a computing cluster to parallelize functions and save time. This is the script we used to run [dada2QAQC.R] using an HPC managed by [SLURM](https://slurm.schedmd.com/documentation.html).

### Amplification bias correction

Following Shelton et al. 2022, we corrected the read count of each species in each sample using two mock mixtures with known proportions of input DNA.

**1. Oorc_master_phyloseq_object.R:** convert output from dada2 pipeline to the correct format for amplification bias correction.

**2. ampbias.QAQC_AOS_AVC.R:** perform amplification bias correction.

**2a. quant_metabar_no_overdispersion_fixed_alpha.stan:** The amplification bias correction Rscript above calls a Bayesian model run in stan using this script.

**3. Oorc_post_ampbias_plots_and_cleanup.R:** Post amplification bias correction, we generate plots showing correction factors and posterior proportions, and convert the corrected data to the phyloseq format for downstream data analysis, adding sample metadata from *new_prey_meta_7.26.22_CE_KP.csv*.

**4. Oorc_post_ampbias_QAQC.R:** In this step, we perform two de-duplication steps: first, we identify *technical* replicates (samples run two or more times), generate plots to compare prey species proportion across replicates, and retain the sample with the highest overall read count. Second, we identify *biological* replicates (two or more samples collected from the same individual on the same day), generate plots to compare species proportion across replicates, and retain the sample with the highest overall read count.

We also perform three filtering steps based on taxonomy: first, we remove ASVs not IDed to species after confirming that these ASVs represent a very small (<<<<1%) of the dataset. Next, we merge all ASVs by species and remove all ASVs that assigned to killer whales. Finally, we remove any species who's read count did not make up at least 1% of the reads in at least 1 sample in the dataset.

### Data analysis


